### PR TITLE
Multiple pipeline editor updates

### PR DESCRIPTION
**Pull request by lresende**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx [hunk 1]
```diff
@@ -570,7 +576,9 @@ export class PipelineEditor extends React.Component<
   }
 
   handleOpen(): void {
+    console.log('>>> handleOpen');
     toArray(this.browserFactory.defaultBrowser.selectedItems()).map(item => {
+      console.log('Opening ==> ' + item.path);
```
Comment (kevin-bates) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx]: This logging seems redundant.  I don't think either of these is necessary, especially when there's already the same log statement happening below if the item is not a directory.

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx [hunk 2]
```diff
@@ -492,6 +487,17 @@ export class PipelineEditor extends React.Component<
     }
   }
 
+  /*
+   * Open node associated notebook
+   */
+  handleOpenNotebook(selectedNodes: any): void {
```
Comment (kevin-bates) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx:513]: Nice refactor.

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx [hunk 3]
```diff
@@ -492,6 +487,17 @@ export class PipelineEditor extends React.Component<
...
+    for (let i = 0; i < selectedNodes.length; i++) {
+      const path = this.canvasController.getNode(selectedNodes[i]).app_data
+        .filename;
```
Comment (kevin-bates) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx:516]: Could we please join this to the line above?  It reads so much easier and was that way prior to the refactor.

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx [hunk 4]
```diff
@@ -278,7 +278,7 @@ export class PipelineEditor extends React.Component<
           contextMenuActionHandler={this.contextMenuActionHandler}
           clickActionHandler={this.clickActionHandler}
           editActionHandler={this.editActionHandler}
-          tipHandler={this.tipHandler}
+          tipHandler={this.tipActionHandler}
```
Comment (ajbozarth) [inline packages/pipeline-editor/src/PipelineEditorWidget.tsx:281]: if it's called `tipHandler` in canvas why are we using a different name?

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx (hunk 3 above)
Comment (ajbozarth) [reply packages/pipeline-editor/src/PipelineEditorWidget.tsx:516]: this is probably a lint fix

**Comment by ajbozarth** [comment]
@kevin-bates adressing your comment about adding `Pipeline` to all the handler names: it is redundant, but it also helps clarify that its a handler for the pipeline editor and not a canvas handler we override. This makes reading the (very large) file slightly easier.

File: packages/pipeline-editor/src/PipelineEditorWidget.tsx (hunk 3 above)
Comment (lresende) [reply packages/pipeline-editor/src/PipelineEditorWidget.tsx:516]: yes, lint... @ajbozarth maybe we could increase the line length :)

Comment (kevin-bates) [reply packages/pipeline-editor/src/PipelineEditorWidget.tsx:516]: FWIW, the backend uses a max-line length of 120.  IMO, limiting lines to 80 characters is too constraining, especially given terminals of the last **30 years**.  If I was using a  [VT100](https://upload.wikimedia.org/wikipedia/commons/9/99/DEC_VT100_terminal.jpg) again, yeah, lines > 80 characters would make things difficult to read.

**Pull request merged**